source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Enhance import-export with OVA
+category: feature
+authors: arik
+wiki_category: Feature
+wiki_title: Features/Enhance import-export with OVA
+wiki_revision_count: 1
+wiki_last_updated: 2017-03-20
+feature_name: Enhance import-export with OVA
+feature_modules: engine, vdsm
+feature_status: planning
+---
+
+# Backgroud
+Open Virtual Appliance (OVA) is a package that contains the different elements of a Virual Machine (VM). OVA is typically a tar archive file that includes the configuration of the VM in the form of the Open Virtualization Format (OVF) as well as virtual disks and other VM-related data.  
+
+OVA is a known format that is used by different organizations and virtualization platforms. Its specification can be found [here](http://www.dmtf.org/standards/ovf). The standard aims at being an ["open, secure, portable, efficient and extensible format for the packaging and distribution of software to be run in virtual machines"](https://en.wikipedia.org/wiki/Open_Virtualization_Format).  
+
+Recently, oVirt was enhanced with support for importing OVAs that were generated by VMware. In addition, oVirt makes an extensive use of the OVF standard to describe VM configuration in, e.g., VM snapshots and exported VMs.  
+
+The ability to package all the VM data, optionally compressed, in one file eases the processes of backing up a VM and porting a VM to a different platform or hypervisor. Unfortunately, the portability is limited by compatibility issues. Different disk formats, drivers, guest-agents, and OVF structure, hinder the ability to import any OVA to any platform. Therefore, different organizations develop their own conversions tools, like Red Hat's [virt-v2v](http://libguestfs.org/virt-v2v.1.html), for converting VMs from other platforms to their platform.   
+
+# Scope
+This feature is about enhancing the import VM and export VM processes in oVirt by making a broader use of the OVA format.  
```

Comment:
Suggested edits:
This feature enhances VM import and export processes in oVirt, by making a broader use of the OVA format.

Note: If "import VM" and "export VM" are actual commands, you can use:
This feature enhances import VM and export VM processes in oVirt, by making a broader use of the OVA format.